tan.json

```diff
@@ -1,7 +1,7 @@
 {
     "id": "tan",
     "summary": "Tangent",
-    "description": "Computes the tangent of `x`. The tangent is defined to be the sine of x divided by the cosine of x.\n\nWorks on radians only.\nThe no-data value `null` is passed through and therefore gets propagated.",
+    "description": "Computes the tangent of `x`. The tangent is defined to be the sine of x divided by the cosine of x.\n\nWorks on radians only.\nThe no-data value `null` is passed through and therefore gets propagated. `NaN` is returned for values where the tangent function is not defined.",
```

Comment:
why not infinity like in #473 as there is a division by zero there?

Reply:
I tried to get infinity in practice (e.g. `Math.tan(Math.PI/2))` in JS and `(math.tan(math.pi / 2)` in Python) but due to floating point inaccuracies it seems mostly impossible to actually get a infinity result. So I left it out, but I guess it would be more accurate to add it...

> The computations follow [IEEE Standard 754](https://ieeexplore.ieee.org/document/8766229) whenever the processing environment supports it. Therefore, *`tan(pi()/2 + multipliy(pi(), n))`* with `n` being any integer results in ±infinity if the processing environment supports it. -infinity for negative values passed to `tan`, +infinity otherwise. If the processing environment does not supports it, an exception is thrown.

Quite lengthy though...

Reply:
> due to floating point inaccuracies it seems mostly impossible to actually get a infinity result.

but then you can't also produce nan as it is currently defined in the description

> `NaN` is returned for values where the tangent function is not defined.


As the description defines "The tangent is defined to be the sine of x divided by the cosine of x", you could get away with it by saying "Infinity is returned for values where the sine of x is zero"

Reply:
I just added the lengthy one from above, that should work.